server/src/main/scala/services/graphstages/Crunch.scala

```diff
@@ -294,4 +293,12 @@ object Crunch {
     val adjustedMinute = minuteInQuestion.addMinutes(-1 * offsetMinutes)
     Crunch.getLocalLastMidnight(MilliDate(adjustedMinute.millisSinceEpoch)).addMinutes(offsetMinutes)
   }
+
+  def listOp[A](op: (A, A) => A)(v1: Iterable[A], v2: Iterable[A]): Iterable[A] = (v1, v2) match {
```

Comment:
this could maybe have a better name?

Reply:
does it belong here as well?

Reply:
yeah! I've refactored & renamed to `reduceIterables`. It now does the actual reduction rather than passing it to `reduce`. And I've added tests around ignoring empty lists

Reply:
As for where it belongs, we don't really have an existing 'general helper functions' area. The crunch object has kind of acted as that place so far